tests/unit/wait.tcl

```diff
@@ -179,33 +179,47 @@ tags {"wait aof network external:skip"} {
                 waitForBgrewriteaof $replica ;# Make sure there is no AOFRW
 
                 $master incr foo
-                assert_equal [$master waitaof 0 1 0] {1 1}
+                assert_equal [$master waitaof 0 1 5000] {1 1}
             }
 
             test {WAITAOF replica copy everysec with AOFRW} {
                 $replica config set appendfsync everysec
 
-                # Need an AOFRW that can be done in a second.
+                # When we trigger an AOFRW, a fsync is triggered when closing the old INCR file,
+                # so with the everysec, we will skip that second of fsync, and in the next second
+                # after that, we will eventually do the fsync.
                 $replica bgrewriteaof
```

Comment:
don't we need a slow AOFRW here too?

Reply:
I want to make a fast AOF, which is completed in one second
in the new code, in the new code, i think we no longer have a race

Reply:
you mean completed in less than one second. so maybe we want tot make this predictable (rather than having a different order in each CI run) and add a waitForBgrewriteaof before the incr?